packages/api/app/index.js

```diff
@@ -12,11 +12,45 @@ const awsServerlessExpress = require('aws-serverless-express');
 const awsServerlessExpressMiddleware = require('aws-serverless-express/middleware');
 
 const { services } = require('@cumulus/aws-client');
+const { getRequiredEnvVar } = require('@cumulus/common/env');
+const { inTestMode } = require('@cumulus/common/test-utils');
 const { MissingRequiredEnvVar } = require('@cumulus/errors');
+const { secretsManager } = require('@cumulus/aws-client/services');
+const Logger = require('@cumulus/logger');
 
 const router = require('./routes');
 const { jsonBodyParser } = require('./middleware');
 
+const log = new Logger({ sender: '@api/index' });
+
+// Load Environment Variables
+// This should be done outside of the handler to minimize Secrets Manager calls.
+const initEnvVarsFunction = async () => {
+  if (inTestMode() && !process.env.INIT_ENV_VARS_FUNCTION_TEST === true) {
```

Comment:
```suggestion
  if (inTestMode() && process.env.INIT_ENV_VARS_FUNCTION_TEST !== true) {
```

I think this should be the condition

Reply:
33b146331